data/debian-distro-info.json

```diff
@@ -123,11 +123,18 @@
   "v12": {
     "codename": "Bookworm",
     "series": "bookworm",
-    "created": "2021-08-14"
+    "created": "2021-08-14",
+    "release": "2023-06-10",
+    "eol": "2026-06-10"
   },
   "v13": {
     "codename": "Trixie",
     "series": "trixie",
-    "created": "2023-08-01"
+    "created": "2023-06-10"
```

Comment:
We should work out why it took us a month to detect this LTS. Did they take a month to update their site?